dev-python/mkdocs-bootstrap/mkdocs-bootstrap-1.1-r1.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+DOCBUILDER="mkdocs"
+
+inherit distutils-r1 docs
+
+DESCRIPTION="Bootstrap theme for MkDocs"
+HOMEPAGE="https://www.mkdocs.org https://github.com/mkdocs/mkdocs-bootstrap"
+SRC_URI="https://github.com/mkdocs/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="dev-python/mkdocs"
```

Comment:
Remind me again was this a DEPEND or BDEPEND? Or perhaps both? Doesn't your eclass handle this?

Reply:
The eclass adds `BDEPEND+=" doc? ( ${DOCDEPEND} )"`, however this package doesn't just need mkdocs to compile the docs. It is a theme for mkdocs, and without mkdocs the theme is useless. Therefore, this should depend on mkdocs even when USE="-doc", and mkdocs should be present both during build time, and runtime.

We had the same conversation the last time I bumped mkdocs and friends: https://github.com/gentoo/gentoo/pull/15150